Client/Frontend/Settings/Brave Today/BraveTodaySettingsViewController.swift

```diff
@@ -48,7 +48,16 @@ class BraveTodaySettingsViewController: TableViewController {
         dataSource.sections = [
             .init(
                 rows: [
-                    .boolRow(title: Strings.BraveToday.isEnabledToggleLabel, option: Preferences.BraveToday.isEnabled)
+                    .boolRow(title: Strings.BraveToday.isEnabledToggleLabel, option: Preferences.BraveToday.isEnabled, onValueChange: { value in
```

Comment:
Is `onValueChange` handler safe without weak or unowned `self` here?

Reply:
Hmm, no I think it'll lead to a retain cycle, will add weak/unowned